src/Datadog.Trace.ClrProfiler.Native/calltarget_tokens.cpp

```diff
@@ -79,7 +81,7 @@ HRESULT CallTargetTokens::EnsureCorLibTokens() {
     unsigned type_buffer;
     auto type_size = CorSigCompressToken(typeRef, &type_buffer);
 
-    auto* signature = new COR_SIGNATURE[runtimeTypeHandle_size + type_size + 4];
+    COR_SIGNATURE signature[signatureBufferSize];
```

Comment:
[Non-blocking] I know we were not doing it before, but should we worry about zeroing out the buffer?
```suggestion
    COR_SIGNATURE signature[signatureBufferSize] {};
```

Reply:
Zeroing has a cost and I was hopping to avoid that.  When we use it we also pass the actual length so that shouldn’t be a problem.